### PR TITLE
Chore: update Reserve Rights token address

### PR DESCRIPTION
**Pull request by larrythecucumber321**
RSR token address is updated. See below recent DEX activity for RSR linked to the new address and the stale activity for the old token.

New:
image
 
Old:
image

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -613,7 +613,6 @@ VALUES
     ("rndr-render-token", "ethereum", "RNDR", "0x6de037ef9ad2725eb40118bb1702ebb27e4aeb24", 18),
     ("rook-keeperdao", "ethereum", "ROOK", "0xfa5047c9c78b8877af97bdcb85db743fd7313d4a", 18),
     ("rpl-rocket-pool", "ethereum", "RPL", "0xb4efd85c19999d84251304bda99e90b92300bd93", 18),
-    ("rsr-reserve-rights", "ethereum", "RSR", "0x8762db106b2c2a0bccb3a80d1ed41273552616e8", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:616]: Hi @larrythecucumber321 , it seems you deleted the token `rsr-reserve-rights` in this PR without adding the one with the new address. Do you want to add the new contract address to `rsr-reserve-rights`?

https://api.coinpaprika.com/v1/coins/rsr-reserve-rights

Comment (larrythecucumber321) [reply models/prices/prices_tokens.sql:616]: Hi @Hosuke, there is an existing (proper) entry for `rsr-reserve-rights` here :) https://github.com/duneanalytics/spellbook/blob/c2a1a0ccb1d6879b5831ee05cdc1920a752814a5/models/prices/prices_tokens.sql#L921

**Review by Hosuke — APPROVED** [review]
This PR removes a duplicate/deprecated record of `rsr-reserve-rights` in prices_tokens.sql.
Another record is here: https://github.com/duneanalytics/spellbook/blob/c2a1a0ccb1d6879b5831ee05cdc1920a752814a5/models/prices/prices_tokens.sql#L921

**Review by dot2dotseurat — APPROVED** [review]

**Comment by dot2dotseurat** [comment]
Ah token prices automation apparently can't handle checks for removing a line, only adding line. I'll edit the grep to handle this case. 

Thanks for finding the dupe @larrythecucumber321 and great username!

**Pull request merged**